build/nuget/uno.winui.runtime-replace.targets

```diff
@@ -87,7 +61,7 @@
 
 	<!-- Force a rebuild on runtime files changes -->
 	<ItemGroup Condition="'$(UnoRuntimeIdentifier)'!=''">
-		<UpToDateCheckInput Include="$(MSBuildThisFileDirectory)/uno-runtime/*" />
+		<UpToDateCheckInput Include="$(MSBuildThisFileDirectory)/../uno-runtime/*" />
```

Comment:
@jeromelaban This file is packaged in buildTransitive, so uno-runtime is one level up.

@ramezgerges This might be why you weren't hitting breakpoints after using NuGet package override. What could have been happening is that the app wasn't rebuilt after modifying the binaries in uno-runtime.